src/FizzWare.NBuilder/Fizzware.NBuilder.csproj

```diff
@@ -27,6 +27,14 @@
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net40'">
     <DefineConstants>NET40;NETFULL</DefineConstants>
   </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(Configuration)'=='Debug'">
+    <DocumentationFile>FizzWare.NBuilder.xml</DocumentationFile>
+  </PropertyGroup>
+  
+  <PropertyGroup Condition="'$(Configuration)'=='Release'">
```

Comment:
Why include a Condition check for the same file? just replace lines 30 -> 37 with 

```
<PropertyGroup>
    <DocumentationFile>FizzWare.NBuilder.xml</DocumentationFile>
  </PropertyGroup>
```

so it's included regardless of the condition. Or is there something i'm miss understanding, here?

Reply:
This was auto-gen'd from when I set the documentation file via the project properties (not csproj directly). I just pushed an update to consolidate this!